libcodechecker/server/client_db_access_handler.py

```diff
@@ -78,11 +161,14 @@ def construct_report_filter(report_filters):
             AND.append(Report.severity == report_filter.severity)
         if report_filter.bugHash:
             AND.append(Report.bug_id == report_filter.bugHash)
+        if report_filter.status is not None:
+            # status value can be 0
```

Comment:
This is not a fully formatted comment. Also, what does `0` mean? Shouldn't we say ".status value can be None, e.g. [some example case here?]"